bindata/v4.0.0/apiservice-cabundle-controller/clusterrole.yaml

```diff
@@ -13,3 +13,16 @@ rules:
   - watch
   - update
   - patch
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
```

Comment:
Is it necessary for the controller to access configmaps across the entire cluster, or only in the `service-ca namespace`?

Reply:
I have added that change after seeing the following error:
```
configmaps "extension-apiserver-authentication" is forbidden: User "system:serviceaccount:openshift-service-ca:apiservice-cabundle-injector-sa" cannot get resource "configmaps" in API group "" in the namespace "kube-system"
```

Reply:
That suggests limiting the scope of authorization to that resource in that namespace rather than granting global access to configmaps.